packages/ui-components/src/components/status/status.tsx

```diff
@@ -0,0 +1,24 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { EStatusType, IStatus } from './status.types';
+import { STATUS_ICON_NAME, STATUS_STATE, STATUS_TEXT } from './status.utils';
+
+@Component({
+	tag: 'kv-status',
+	styleUrl: 'status.scss',
+	shadow: true
+})
+export class KvStatus implements IStatus {
```

Comment:
Naming can be a bit confusing here, should we change this to `KvTagStatus` or `KvStatusTag`? 

I believe this naming schema can also be confusing in the props (i.e `type`, where we're actually defining the state of this component) but this is debatable.